annis-visualizers/src/main/antlr4/annis/visualizers/htmlvis/HTMLVisConfig.g4

```diff
@@ -34,18 +34,21 @@ NEWLINE : '\n';
 COMMENT : '#' ~('\n')+ -> skip;
 ID: [a-zA-Z0-9\_\-*?]+;
 TXT : (.)+?;
+TEMPVALUE :'%%value%%';
+TEMPANNO: '%%anno%%';
 
 innervalue: ~(QUOTE)+;
 value : QUOTE innervalue QUOTE;
-
-innertype: ~(QUOTE)+;
-
+temp: TEMPVALUE|TEMPANNO;
+innertype: ~(QUOTE|TEMPVALUE|TEMPANNO)+;
+innerhtmltemp: (innertype* temp innertype*)+; 
 innermeta: ~(QUOTE|WS|NEWLINE)+;
 
 type
   : VALUE # typeValue
   | ESCAPED_VALUE # typeEscapedValue
   | ANNO # typeAnno
+  | QUOTE innerhtmltemp QUOTE # typeHtmlTemp
```

Comment:
actually I thought the "~" in " innertype: ~(QUOTE|TEMPVALUE|TEMPANNO)+  " means that innertype doesn't allow QUOTE or TEMPVALUE or TEMPANNO, no?